h/views.py

```diff
@@ -26,14 +26,19 @@ def annotation(context, request):
     title = 'Annotation by {user} on {title}'.format(
         user=context['user'].replace('acct:', ''),
         title=context['document']['title'])
```

Comment:
While you're here would you also fix this line, not all annotations are guaranteed to have the document property so we need a fallback if it's missing.